Sources/Defaults/SwiftUI.swift

```diff
@@ -4,7 +4,7 @@ import Combine
 
 @available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
 extension Defaults {
-	final class Observable<Value: Codable>: ObservableObject {
+	final class Observable<Value: DefaultsSerializable>: ObservableObject {
```

Comment:
```suggestion
	final class Observable<Value: Serializable>: ObservableObject {
```

Probably in other places too.